### DISCUSSION TITLE
Device Setting Tab - K Factor from Bambu Studio

### DISCUSSION THREAD

**Discussion by Ornytus** [OrcaSlicer/OrcaSlicer#508] [Q&A]
I decided to finally start using the Soft Fever fork of Bambu Studio as I feel I am missing out!

Checking out the calibration tools, I was wondering if the k factor in the device tab (screenshot for reference attached) that I set previously in Bambu Studio will override the PA setting applied in the filament setting in Orca Slicer? Am I good to leave it as is, should I zero it and only have a value entered in the filament settings?

Thanks for any help

![Screenshot 2023-03-17 190413](https://user-images.githubusercontent.com/96362434/226074936-b36b8da9-b3fa-4103-99f8-cced33692500.png)

**Comment by SoftFever** [comment]
The PA value you set in filament setting will override the value

**Reply by Ornytus** [reply]
Thanks for the info, and appreciate your work on this project, what you’ve put together is awesome!
